src/contexts/AuthContextProvider.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+import { ReactNode, useEffect, useState } from "react";
+import { AuthContext } from "./AuthContext";
+import axios from "axios";
+import { setCookie, getCookie, deleteCookie } from "cookies-next";
+
+export function AuthProvider({ children }: { children: ReactNode }) {
+  const [isAuth, setIsAuth] = useState(false);
+
+  useEffect(() => {
+    const token = getCookie("token");
```

Comment:
its better practice to dont use hardcode values. in your constants.ts file create varivale const COKIES_TOKEN = 'token'  -> const token = getCookie(COKIES_TOKEN);  or create enum